components/pages/page/Page.tsx

```diff
@@ -20,14 +20,17 @@ export function Page({ data }: { data: PagePayload }) {
 
   return (
     <div>
-      {/* Header */}
-      <Header title={title} description={overview} />
+      <div className="mb-14">
+        {/* Header */}
+        <Header title={title} description={overview} />
 
-      {/* Body */}
-      {body && <PortableText components={components} value={body} />}
+        {/* Body */}
+        {body && <PortableText components={components} value={body} />}
 
-      {/* Workaround: scroll to top on route change */}
-      <ScrollUp />
+        {/* Workaround: scroll to top on route change */}
+        <ScrollUp />
+      </div>
+      <div className="absolute left-0 w-screen border-t" />
```

Comment:
Looks to me that this should probably be in the footer :)

Reply:
But it's not supposed to be on the front page

Reply:
I know that it doesn't have it in the design (on the home page) but it looks like it exists on both pages and on project details pages 🤔 hence why I felt it made sense to either have them on all or none, feels kinda strange to duplicate this line

Reply:
I had it before on all pages, but then Rob removed it on all pages - whatever you guys want to do, I am fine with. But this is correct according to figma

Reply:
@robinpyon booooo 
Alright, the line stays 😔